docs/rules/no-unmodified-loop-condition.md

```diff
@@ -29,6 +29,8 @@ If a reference is inside of a dynamic expression (e.g. `CallExpression`,
 Examples of **incorrect** code for this rule:
 
 ```js
+/*eslint no-unmodified-loop-condition: "error"*/
+
 while (node) {
     doSomething(node);
 }
```

Comment:
This doesn't seem to work in the demo (at least when I try), maybe the variables have to be declared?

@yeonjuan if this really should be fixed, what do you think about doing that in this PR or in another PR?

Reply:
> if this really should be fixed, what do you think about doing that in this PR or in another PR?
@mdjermanovic   I think doing in this PR nice. I fixed it. Thanks 👍

Reply:
Thanks! The example with for-loop also doesn't work, but that seems to be a more complex problem for another PR. I'm not sure what was the expected behavior.